packages/i18n/src/locales/en.json

```
@@ -336,6 +331,7 @@
         "video": "Video",
         "weekly": "Weekly Updates",
         "wg": "Working Groups",
+        "migrations": "Migrations Guides",
```

Grammar: I suppose this could be correct because we offer many per version - I think in casual conversation folks would think this is an error
```suggestion
        "migrations": "Migration Guides",
```